src/languages/en.ts

```diff
@@ -2798,6 +2798,7 @@ const translations = {
             formLabel: 'View PDF',
         },
         attachmentNotFound: 'Attachment not found',
+        tryAgain: 'Try again',
```

Comment:
May be I missed it somewhere but the text in the design copy seems to be `Retry` but we're using `Try again`?

Reply:
Confirmed on the [thread](https://expensify.slack.com/archives/C05LX9D6E07/p1757010278473289?thread_ts=1748013453.884369&cid=C05LX9D6E07), we should be using `Retry`.